lesson-02-assignment.rb

```diff
@@ -0,0 +1,41 @@
+#QUESTION 1
+
+class_ids = CodingClass.class_ids
+class_ids.each {|class_id|
+    Course.create(coding_class_id: class_id, trimester_id: '8', max_enrollment: '25')}
```

Comment:
always be conscious of types, the  trimester_id is an integer in the migration but you are using numeric string.
```suggestion
    Course.create(coding_class_id: class_id, trimester_id: 8, max_enrollment: 25)}
```
well done.